Code/Editor/Util/FileUtil.cpp

```diff
@@ -305,6 +305,12 @@ void CFileUtil::EditTextFile(const char* txtFile, int line, IFileUtil::ETextFile
     ExtractFile(fullPathName);
     QString cmd = Platform::MakePlatformFileEditString(fullPathName, line);
 
+    if (fileType == IFileUtil::FILE_TYPE_SYSTEM)
+    {
+        QDesktopServices::openUrl(QUrl::fromLocalFile(fullPathName));
+        return;
+    }
+
```

Comment:
This API is for using the pre-defined editors that are specified by the user. The `FILE_TYPE_SCRIPT` type is typically meant to be some kind of text editor so users can modify scripts (e.g. python, lua). If the user specified the LUAIde as their text editor, they wouldn't be able to open python files in there either, so I see that more of a misconfiguration.

I could potentially see this change being reduced to just calling `QDesktopServices::openUrl(QUrl::fromLocalFile(CLogFile::GetLogFileName()));` in `CCryEditApp::OnFileEditLogFile()`, but I don't think the file edit API should be modified for this.

Reply:
done.
Please review the code.

Reply:
Thank you!